spec/ruby/core/io/binread_spec.rb

```diff
@@ -44,4 +44,14 @@
   it "raises an Errno::EINVAL when not passed a valid offset" do
     -> { IO.binread @fname, 0, -1  }.should raise_error(Errno::EINVAL)
   end
+
+  ruby_version_is "3.3" do
+    it "warns about deprection given a path with a pipe" do
+      cmd = "|date"
+      platform_is(:windows) { cmd = "|date /t" }
```

Comment:
`echo`  command should work even on Windows.
```suggestion
      cmd = "|echo ok"
```

Reply:
OK, I've updated the PR to use `|echo ok` on all platforms for all the new specs.